.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Each line is a file pattern followed by one or more owners.
```

Comment:
We don't need this file unless we want to get tagged if someone tries to change this, it won't actually prevent changes.